README.md

```diff
@@ -195,11 +218,11 @@ _Optional_ - custom tagged template literal.
 
 When using GraphQL with JavaScript or TypeScript projects, it is common to use the `gql` tagged template literal to write out operations. Apollo tools look through your files for the `gql` tag to extract your queries, so if you use a different template literal, you can configure it like so:
 
-```js
-module.exports = {
-  client: {
-    tagName: "graphql", // highlight-line
-    service: ...
```

Comment:
![image](https://github.com/user-attachments/assets/00f31731-80d7-4784-84b0-d7a35a1e4ddc)
This actually rendered out in the [Marketplace](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo), so I'm removing it